internal/ad/ad.go

```diff
@@ -304,10 +303,7 @@ func (ad *AD) GetPolicies(ctx context.Context, objectName string, objectClass Ob
 		// check assetsSrc exists
 		_, err = os.Stat(assetsSrc)
 		if errors.Is(err, fs.ErrNotExist) {
-			if err := os.Remove(filepath.Join(assetsSrc + ".db")); err != nil {
-				return err
-			}
-			return nil
```

Comment:
I really can imagine the linter going crazy on us by analyzing this code: "really, guys???"